### PR TITLE
Closed Raytheon

### PR DESCRIPTION
**Pull request by DEVSOG12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -351,7 +351,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
...
 | [Crusoe](https://jobs.lever.co/crusoeenergy/16d9dc19-74a5-4c32-9ead-e3e221f4eaca) | San Francisco, CA | University / Software Engineer (Intern) |
 | [Farm Credit Services of America](https://careers.fcsamerica.com/job/16636430/) | Omaha, NE | Web Software Systems Engineer (Intern Summer 2023) |
-| Raytheon | Varies by Role | [UP 2023 Software Engineer Internships](https://careers.rtx.com/global/en/job/RAYTGLOBAL01567022EXTERNALENGLOBAL/UP-2023-Software-Engineer-Internships), [2023 Software Engineer Summer Intern](https://careers.rtx.com/global/en/job/RAYTGLOBAL01569607EXTERNALENGLOBAL/2023-Software-Engineer-Summer-Intern) (US Citizenship Required for Both) |
+| Raytheon | Varies by Role | **🔒 Closed 🔒** [UP 2023 Software Engineer Internships], [2023 Software Engineer Summer Intern] (US Citizenship Required for Both) |
```
Comment (ewang2002) [inline README.md]: Are you sure the UP 2023 SWE Internship is open? 

image

It is still open for me. The other internship is closed, though.

**Review by ewang2002 — COMMENTED** [review]
See my comment. Can you send me a screenshot of what you're seeing?

File: README.md (hunk 1 above)
Comment (DEVSOG12) [reply README.md]: Sorry, it's only the 2023 Software Engineer Summer Intern

Comment (DEVSOG12) [reply README.md]: I've fixed it

**Review by ewang2002 — APPROVED** [review]
Looks good. Thanks!!

**Pull request merged**